### ISSUE TITLE
Quite a few notices on an upgraded site for forum

### ISSUE THREAD

**Issue by JoakimFalk** [PHPFusion/PHPFusion#631]
Are we missing some tables still or is something else going on...
I am out of time for today so I post this and we see later.

templates/forum_main.php Undefined index: permissions Line: 273
templates/forum_main.php Undefined index: permissions Line: 271
templates/forum_main.php Undefined index: permissions Line: 272
templates/forum_main.php Undefined index: item Line: 199
templates/forum_main.php Undefined index: permissions Line: 269
templates/forum_main.php Undefined index: permissions Line: 270

**Comment by meangczac** [comment]
L273 - 
<span>".sprintf($locale['forum_perm_download'], $info['permissions']['can_download_attach'] == TRUE ? "<strong  class='text-success'>".$locale['can']."</strong>" : "<strong class='text-danger'>".$locale['cannot']."</strong>")."</span><br/>

is 

$this->forum_info['permissions']['can_download_attach'] = iMOD || ($forum_data['forum_allow_attach'] == TRUE && checkgroup($forum_data['forum_attach_download'])) ? TRUE : FALSE;

L271:
<span>".sprintf($locale['forum_perm_create_poll'], $info['permissions']['can_create_poll'] == TRUE ? "<strong  class='text-success'>".$locale['can']."</strong>" : "<strong class='text-danger'>".$locale['cannot']."</strong>")."</span><br/>

is

$this->forum_info['permissions']['can_create_poll'] = $forum_data['forum_allow_poll'] == TRUE && (iMOD || (checkgroup($forum_data['forum_poll']) && $forum_data['forum_lock'] == FALSE)) ? TRUE : FALSE;

L272:
<span>".sprintf($locale['forum_perm_upload'], $info['permissions']['can_upload_attach'] == TRUE ? "<strong  class='text-success'>".$locale['can']."</strong>" : "<strong class='text-danger'>".$locale['cannot']."</strong>")."</span><br/>

is

$this->forum_info['permissions']['can_upload_attach'] = $forum_data['forum_allow_attach'] == TRUE && (iMOD || checkgroup($forum_data['forum_attach'])) ? TRUE : FALSE;

L269:
<span>".sprintf($locale['forum_perm_access'], $info['permissions']['can_access'] == TRUE ? "<strong class='text-success'>".$locale['can']."</strong>" : "<strong class='text-danger'>".$locale['cannot']."</strong>")."</span><br/>

is

$this->forum_info['permissions']['can_access'] = (iMOD || checkgroup($forum_data['forum_access'])) ? TRUE : FALSE;

L270:
<span>".sprintf($locale['forum_perm_post'], $info['permissions']['can_post'] == TRUE ? "<strong class='text-success'>".$locale['can']."</strong>" : "<strong class='text-danger'>".$locale['cannot']."</strong>")."</span><br/>

is

$this->forum_info['permissions']['can_post'] = (iMOD || (checkgroup($forum_data['forum_post']) && $forum_data['forum_lock'] == FALSE)) ? TRUE : FALSE;

Basically the whole set of forum access checks failed.

And Line 199 most importantly:

$data = $info['item'][$_GET['forum_id']];

That forum ID is not available in the whole item list.

**Comment by meangczac** [comment]
If have missing column, his forum will fail now. Check for any SQL error logs.

**Comment by JoakimFalk** [comment]
Server hiccup
